pkg/cmd/ui/tty.go

```diff
@@ -47,3 +47,14 @@ type noopWriter struct{}
 var _ io.Writer = noopWriter{}
 
 func (w noopWriter) Write(data []byte) (int, error) { return len(data), nil }
+
+// Used for testing whether TTY writes correct output to stdout/stderr
+func NewMockTTY(debug bool, stdout, stderr io.Writer) tty {
```

Comment:
this isnt really a mock though. it's `tty` class that allows custom stdout, stderr.

Reply:
Renamed to `NewCustomWriterTTY`